cli/main.go

```diff
@@ -26,13 +27,32 @@ func execute() error {
 	return cmd.ExecuteContext(ctx)
 }
 
-var version = "0.0.0-dev"
+var (
+	version           = "0.0.0-dev"
+	runtimeHandler    = "contrast-cc"
+	launchDigest      = "000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000"
+	genpolicyVersion  = "0.0.0-dev"
+	containerVersions = ""
+)
 
 func newRootCmd() *cobra.Command {
+	// build the versions string
+	var versionsBuilder strings.Builder
+	versionsBuilder.WriteString(fmt.Sprintf("%s:\n\n", version))
+	versionsBuilder.WriteString(fmt.Sprintf("\truntime handler: %s\n", runtimeHandler))
+	versionsBuilder.WriteString(fmt.Sprintf("\tlaunch digest: %s\n", launchDigest))
+	versionsBuilder.WriteString(fmt.Sprintf("\tgenpolicy version: %s\n", genpolicyVersion))
+	for _, container := range strings.Split(containerVersions, ",") {
+		containerSplit := strings.Split(container, "@")
```

Comment:
The released images are also tagged with a version, so the images are of the form `.../<container-name>:v1.2.3@sha256:abc...`, so in that case I would split at the first `:`.

Reply:
Since images are now simply printed, I ignored that.